addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -193,16 +195,32 @@ public void close() {
      *
      * @since 7.2
      */
-    protected void abortOldUploads() throws IOException {
+    protected void abortOldUploads() {
+        ConfigurationService configurationService = Framework.getService(ConfigurationService.class);
+        if (configurationService == null) {
```

Comment:
in which case does it happen?

Reply:
It's not something I expect to happen, but I want to be robust in backports (and maybe some pure unit tests).